loopy/codegen/loop.py

```diff
@@ -274,7 +274,7 @@ def set_up_hw_parallel_loops(codegen_state, schedule_index, next_func,
             and other_iname != iname
             and any(_tag.key == tag.key
                     for _tag in kernel.iname_tags(other_iname)
-                    if _tag))]
+                    if isinstance(_tag, InameImplementationTag)))]
```

Comment:
Is this equivalent to `for _tag in kernel.iname_tags_of_type(other_iname, InameImplementationTag)`?

Reply:
Yep, 4e988f3.